### ISSUE TITLE
Feature request: Scale Spritesheets

### ISSUE THREAD

**Issue by BenHerbst** [hippogamesunity/PixelStudioHub#116]
I use this app now for some months and also bought pro. It is the best Pixel art editor for Android tablets, and it is a great and modern workflow. I do game developing with high pixel ui and low pixel pixel art. So I scale my art 4 times, from 16 pixels to 64 as it would be too small with high pixel ui else. Why can't I scale my spritesheets, but single images? I want to save my art scaled by 4 times as spritesheet in png. Why are the scale options grayed out for spritesheet?

**Comment by hippogamesunity** [comment]
Hi, in most cases, scaling sheet results in exceeding max image size. You can open a sheet and save it with upscaling if needed.

**Comment by BenHerbst** [comment]
Yes, but it would be a better workflow. The scale buttons could only be grayed out, if the size would be too big.
